draft-irtf-cfrg-frost.md

```diff
@@ -695,6 +695,27 @@ The value of the contextString parameter is "FROST-RISTRETTO255-SHA512".
     the output to a Scalar as described in {{!RISTRETTO, Section 4.4}}.
   - H3(m): Implemented by computing H(contextString \|\| "digest" \|\| m).
 
+## FROST(edwards448, SHAKE256)
+
+This ciphersuite uses edwards448 for the Group and SHA256 for the Hash function `H`
```

Comment:
So Edwards448 by definition has a cofactor. Should we clarify that we are still trying to do these Schnorr signatures in the prime order subgroup, not 'just' the composite-order group defined by the whole elliptic curve?

Reply:
That's not a bad idea. Can we do this in a followup PR? (And would you be willing to do that?)